packages/cron/src/lib/email.js

```diff
@@ -44,24 +44,24 @@ export class EmailService {
    * @param {object} opts
    * @param {import('@web3-storage/db/db-client-types').UserStorageUsedOutput} opts.user
    * @param {string} opts.email
-   * @param {number} opts.numberOfDays
-  * @returns void
+   * @param {number} opts.secondsSinceLastSent
+   * @returns void
    */
   async sendEmail ({
```

Comment:
I wonder if we should be agreeing on the interface and how we could pass `context` to the email template here so that we could start creating it as part of the `PR`.